enzyme/Enzyme/MLIR/enzymemlir-translate/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ target_link_libraries(enzymemlir-translate
   MLIRSPIRVDialect
   MLIRTranslateLib
   MLIRSupport
+  "ZLIB::ZLIB"
```

Comment:
is this needed?

Reply:
cmake build fails without that, so for now I just added it as a quick fix